### ISSUE TITLE
Identify partner team for layout foundation work

### ISSUE THREAD

**Issue by sarahill** [Shopify/polaris#200]

**Comment by sarahill** [comment]
[Partner criteria](https://www.figma.com/file/W3dGfz3RsiwO6h9IHub85b/Partner-criteria?node-id=0%3A1) that Jared started

**Comment by sarahill** [comment]
Yesenia and Kalv recommended working with Genevieve on Deliver to see how we can work together

**Comment by sarahill** [comment]
Started a working doc with info/guidance on [forming strategic partnerships ](https://docs.google.com/document/d/1_6061-az0-vKVl82EMjj0SdGTQMLZodI2yV9hFUVLhc/edit#heading=h.ej5b62zdpznn)

**Comment by sarahill** [comment]
Chatted with the team working on Linkpop. Here's the [recording](https://drive.google.com/file/d/1eRaKdWCzEFe-IaI6aAOQRignb2mI_wYY/view?usp=sharing) from the session along with [notes](https://docs.google.com/document/d/1MHriP4G4O-zkOZ5zxaxCg2l-PAkZaxn-hX3wM6S-6_Q/edit#).

For next steps, Chloe is gonna chat with devs on our team to get some thoughts on next steps and if it makes sense to partner with this team now. We'll touch base and come up with recommendations that we'll share with Jared, Alex, Yuraima, and Yesenia for next steps for partnering.

Some initial thoughts on my side:
- There is an opportunity here to up-level the quality of mobile web
- Mobile web has been something Shopify has talked about improving for a while and some teams are feeling the pain now. If this is still important to Shopify that could be a way to show impact: approaching the system from a true mobile first perspective.

We're also talking to the Deliver team on Thursday.

**Comment by sarahill** [comment]
After chatting with more folks from our team the consensus is based on the components they've identified, their tight timeline, and where we’re at with the foundations work, we believe that contributing to polaris-react will be the best option for their team right now.

tl;dr: This sounds like a great partnership opportunity for a future cycle. Let’s table for now, circle back when tokens and docs are ready, and focus right now on finding a partner with layout needs primarily.

**Comment by sarahill** [comment]
Chatted with Deliver team today. Here's the [recording](https://drive.google.com/file/d/1t6z-RwGZ170yjVyBVRQ_l0sFO2LcUTvQ/view) along with the [notes](https://docs.google.com/document/d/1MHriP4G4O-zkOZ5zxaxCg2l-PAkZaxn-hX3wM6S-6_Q/edit#).

**Next steps:**
- Alex will share the adoption plan with the Deliver team to get their thoughts and feedback. We can adjust the plan with them.
- Sergey will set up some time with our team in the next week or so to go over the components they've chosen to focus on. We'll identify the foundational elements within those components together.
